exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -238,17 +238,22 @@ func logToCWLog(resourceAttrs map[string]interface{}, log plog.LogRecord, config
 		var metadata emfMetadata
 		bodyString := log.Body().AsString()
 		err = json.Unmarshal([]byte(bodyString), &metadata)
+		switch {
```

Comment:
Our linter wants a switch instead of else if

Reply:
Yeah it seems to do that whenever there are 3+ conditionals.